concent_api/core/admin.py

```diff
@@ -18,22 +18,25 @@ class ActivePassiveDownloadsStateFilter(admin.SimpleListFilter):
 
     def lookups(self, request, model_admin):
         return (
-            ('active', 'Active'),
-            ('passive', 'Passive'),
-            ('active_or_downloads', 'Active or with active downloads')
+            ('active', 'Active and not timed out '),
```

Comment:
I would rephrase it a little bit to make it more clear and match what is in querysets:
"Not timed out active"
"Passive or timed out active"
"Not timed out active or active with pending downloads"

Reply:
Done.